agent/ecscni/types.go

```diff
@@ -119,16 +119,14 @@ type ENIConfig struct {
 	CNIVersion string `json:"cniVersion,omitempty"`
 	// ENIID is the id of ec2 eni
 	ENIID string `json:"eni"`
-	// IPV4Address is the ipv4 of eni
-	IPV4Address string `json:"ipv4-address"`
-	// IPV6Address is the ipv6 of eni
-	IPV6Address string `json:"ipv6-address,omitempty"`
+	// IPAddresses contains the ip addresses of the ENI.
```

Comment:
Not a big deal, but it would be nice for this struct to follow the same order as the actual ecs-eni netconf struct fields: eniid, macaddress, ipaddresses, gatewayipaddresses, blockinstancemetadata.

Reply:
updated the order here and the one in netconfig.go